lib/sles4sap_publiccloud.pm

```diff
@@ -76,12 +76,18 @@ our @EXPORT = qw(
     All commands are executed through C<sudo>.
     If 'runas' defined, command will be executed as specified user,
     otherwise it will be executed as root.
+
+=over 1
+
+=item B<CMD> - command string to be executed remotely
+
+=back
 =cut
 
 sub run_cmd {
     my ($self, %args) = @_;
-    croak("Argument <cmd> missing") unless ($args{cmd});
-    croak("\$self->{my_instance} is not defined. Check module Description for details") unless $self->{my_instance};
+    croak "Argument <cmd> missing" unless $args{cmd};
```

Comment:
I would avoid these kind of changes unless there is a real reason for doing this. 
Both usages are valid, readable, understandable. 
Personally I am even in favor using brackets since they clearly mark section with arguments and it is more friendly towards people coming from python.